nio/schemas.py

```diff
@@ -22,6 +22,7 @@
 RoomRegex = "^!.+:.+$"
 UserIdRegex = "^@.*:.+$"
 EventTypeRegex = r"^.+\..+"
+EventIdRegex = r"^\$.+:.+$"
```

Comment:
This will fail on newer room versions, I think we used to have such a regex but it was removed for this reason.

More info [here](https://matrix.org/docs/spec/rooms/v3#id1).

Reply:
Oh shoot I didn't realise there was a new event ID format. Are you okay with a regex that matches either format then?

Reply:
A regex that matches any string should be ok, the spec even suggests to treat them as opaque strings.